src/runtime_src/core/pcie/driver/linux/xocl/subdev/pmc.c

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (C) 2020 Xilinx, Inc. All rights reserved.
+ *
+ * Authors: David Zhang <davidzha@xilinx.com>
+ *
+ * This software is licensed under the terms of the GNU General Public
+ * License version 2, as published by the Free Software Foundation, and
+ * may be copied, distributed, and modified under those terms.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "../xocl_drv.h"
+
+#define	PMC_ERR(pmc, fmt, arg...)	\
+	xocl_err(&(pmc)->pmc_pdev->dev, fmt "\n", ##arg)
+#define	PMC_WARN(pmc, fmt, arg...)	\
+	xocl_warn(&(pmc)->pmc_pdev->dev, fmt "\n", ##arg)
+#define	PMC_INFO(pmc, fmt, arg...)	\
+	xocl_info(&(pmc)->pmc_pdev->dev, fmt "\n", ##arg)
+#define	PMC_DBG(pmc, fmt, arg...)	\
+	xocl_dbg(&(pmc)->pmc_pdev->dev, fmt "\n", ##arg)
+
+#define	PMC_ERR1_STATUS_MASK	(1 << 24)
+#define	PMC_ERR_OUT1_EN_MASK	(1 << 24)
+#define	PMC_POR1_EN_MASK	(1 << 24)
+#define	PMC_ERR_OUT1_MASK	0x20
```

Comment:
A nit to the Macro name. It would make the code more readable if we name PMC_ERR_OUT1_MASK as PMC_REG_ERR_OUT1_MASK ? (just to differentiate from PMC_ERR1_STATUS_MASK)

Reply:
fixed.